### PR TITLE
[Merged by Bors] - fix: use new variables in smartmodule-publish workflow

### PR DESCRIPTION
**Pull request by ozgrakkurt**

### CODE REVIEW

File: .github/workflows/smartmodule-publish.yml [hunk 1]
```diff
@@ -50,8 +48,8 @@ jobs:
         run: fluvio install smdk
       - name: Fluvio Login
         run: |
-          fluvio cloud login --email ${{ inputs.target_prod && secrets.HUB_USER_EMAIL || secrets.DEV_HUB_USER_EMAIL }} \
-          --password ${{ inputs.target_prod && secrets.HUB_USER_PASSWORD || secrets.DEV_HUB_USER_PASSWORD }} \
+          fluvio cloud login --email ${{ secrets.ORG_HUB_SA_NAME }} \
+          --password ${{ !inputs.target_prod && secrets.ORG_HUB_SA_PASSWD_DEV || secrets.ORG_HUB_SA_PASSWD_PROD }} \
```
Comment (digikata) [inline .github/workflows/smartmodule-publish.yml:52]: The passwords and PROD and DEV are swapped.  Looks good otherwise

**Review by digikata — CHANGES_REQUESTED** [review]

File: .github/workflows/smartmodule-publish.yml (hunk 1 above)
Comment (ozgrakkurt) [reply .github/workflows/smartmodule-publish.yml:52]: should be ok because of the `!`

**Review by sehz — APPROVED** [review]

File: .github/workflows/smartmodule-publish.yml (hunk 1 above)
Comment (digikata) [reply .github/workflows/smartmodule-publish.yml:52]: Oh wait, why invert inputs.targets_prod?

**Comment by sehz** [comment]
Generally it would be easier to test workflow in specific repo and then abstract into a common reusable

**Comment by ozgrakkurt** [comment]
> Generally it would be easier to test workflow in specific repo and then abstract into a common reusable

makes sense, I'll do it like that next time

**Pull request closed without merging**